requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
-pandas==1.4.1
+pandas==1.4.2
```

Comment:
There are also other files where we have to change the version in, you can check the previous PR for the full list of them #4235

Reply:
will do, thanks!